src/health/probe.go

```diff
@@ -2,13 +2,14 @@ package health
 
 import (
 	"fmt"
+	"net/http"
 	"net/url"
 	"strings"
 )
 
 type Probe struct {
 	Exec             *ExecProbe `yaml:"exec,omitempty"`
-	HttpGet          *HttpProbe `yaml:"http_get,omitempty"`
+	Http             *HttpProbe `yaml:"http,omitempty"`
```

Comment:
This breaks backward compatibility.
Do you think we can have both `http` and `http_get` fields at the same time?
Even nicer would be to use the deprecation warning:
https://github.com/F1bonacc1/process-compose/blob/main/src/types/deprecation.go

Reply:
Thanks, will do. Can you give an example of how this is done?

Reply:
Certainly.
Check out this commit as an example:
https://github.com/F1bonacc1/process-compose/commit/45fceb3abfdf36e8205d2322806ea16be23d6a5e

I mistakenly used `on-failure` instead of `on_failure`, once discovered I kept both of them:
1. I issued a warning during the first month after deprecation.
2. Error + 5s sleep during the second month
3. Exit 1 after that.

I would go ahead and add the `deprecationCheck` to `validators.go` and assign the `http_get` values (if exist) to `http` (if not defined) in `mutators.go`.

If that's too many steps, let me know and I will add this content.

Reply:
IMO in case both of them are defined you can fail the loading in `loader.validators.go`.
Please let me know if you need any help with this feature.